src/fielddef.ts

```diff
@@ -112,14 +112,20 @@ export function cardinality(fieldDef: FieldDef, stats, filterNull = {}) {
     (stat.missing > 0 && filterNull[type] ? 1 : 0);
 }
 
-export function title(fieldDef: FieldDef) {
+export function title(fieldDef: FieldDef, showUnit: boolean) {
   if (isCount(fieldDef)) {
     return COUNT_DISPLAYNAME;
   }
+
+  var unit = '';
```

Comment:
Let me know if this is good style. I can try to do something more clean, or if you have any suggestions